pkg/config/config_template.yaml

```diff
@@ -611,6 +611,13 @@ api_key:
     #
     # ad_identifier: snmp
 
+    ## @param loader - string - optional
+    ## Check loader to use. Available loaders:
+    ## - core: will use corecheck SNMP integration
+    ## - python: will use python SNMP integration
+    #
+    # loader: core
```

Comment:
Is it aligned with the default in `datadog.yaml` ?

Reply:
I'm not sure to understand, `loader: core` mentioned is an example of value the user can set (the one we suggest). It's not the default value. Currently the default value is empty string, and the normal loader order will be used (Python snmp integration have a higher precedence, so it will be used).

Reply:
My bad! I was talking about the `conf.yaml` file (check instance config). 
We may just mention the default.